site/en/docs/crux/release-notes/index.md

```diff
@@ -43,6 +43,23 @@ The CrUX dataset on BigQuery is generally updated on the second Tuesday of every
 
 In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release Announcements.
 
+## 202306
+
+[Announcement](https://groups.google.com/a/chromium.org/g/chrome-ux-report-announce/c/?????)
```

Comment:
Reminder to self to update before merging

Reply:
**YOU ARE REMINDED**

Reply:
**I HAVE UPDATED IT**